packages/lib-ethers/utils/deploy.ts

```diff
@@ -11,14 +11,12 @@ import {
 interface IOwners {
   feePool: string
   bammOwner: string
-  legalEntity: string
   governorBravo: string
 }
 
 const contractOwners: IOwners = {
   feePool: "0x7095F0B91A1010c11820B4E263927835A4CF52c9",
-  bammOwner: "0x0000000000000000000000000000000000000001",
-  legalEntity: "0xf642Bd6A9F76294d86E99c2071cFE2Aa3B61fBDa",
+  bammOwner: "0x788c26aB09728A2f17F2372a4D65CcA184940E83",
```

Comment:
What's this address?

Reply:
It's a 2 out of 2 multi-sig contract address, being one address of B. protocol contract, and the other one set for our side with 5 committee volunteers members. More info can be seen at: https://discord.com/channels/866378471868727316/1157795257342230598/1157947513605734410